tests/installation/addon_products_sle.pm

```diff
@@ -17,10 +17,16 @@ use testapi;
 use utils 'addon_license';
 
 sub run() {
+    if (check_var('VERSION', '15')) {    # SLE 15 has unsigned file errors, workaround them - rbrown 04/07/2017
+        while (check_screen('sle-15-unsigned-file')) {
+            record_soft_failure 'bsc#1047304';
+            send_key 'alt-y';
+        }
+    }
```

Comment:
same as above, use multi-tag assert_screen to save the 30 seconds (on every run)